### ISSUE TITLE
stringify object to pass as `functions` option

### ISSUE THREAD

**Issue by dtothefp** [jtangelder/sass-loader#155]
maybe this is an issue I should put on `loader-utils` but for the life of me I can't figure out how to stringify an object to pass as the `functions` option to `node-sass`.  Seems like it would be easier for things like this if you would do something like check `this.options.sass`:

```js
let functions = {
  'image-url($filename)': function(filename) {
    var imageUrl = process.env.NODE_ENV === 'development' ?
      '' :
      '';//enter CDN path here
    var fileName = filename.getValue();

    var imagePath = 'url("' + imageUrl + '/' + fileName + '")';
    /*eslint-disable */
    return new sass.types.String(imagePath);
    /*eslint-enable */
  }
}

`sass-loader?functions=${somehowStringify(functions))}`
```

**Comment by jorrit** [comment]
PR #152 would help in your case I think.

**Comment by jhnns** [comment]
Yep, #152 solves it. I just merged the PR, but we need tests :grinning:.

**Comment by jhnns** [comment]
#152 shipped with `3.1.0`. New preferred way for passing options is now via a property on your webpack config. See https://github.com/jtangelder/sass-loader#sass-options

**Comment by dtothefp** [comment]
@jhnns awesome...much appreciated
